alexsg2/Practice_10k_Music_App
# Setting Up Test

# Why
Adding tests helps provide a safety net for the application. As we add more and have multiple people working on the same parts of the app being able to run the tests to confirm everything is still working is valuable.


# What

I added snapshot testing which takes a "snapshot" of the html and css. If new css or html is added to the application then the snapshot tests with detect this.

I also added unit tests which test the test input fields and buttons on the start page.

# How to test

Fetch this branch are reset hard
Install packages
In terminal: `yarn run test`
All tests should pass
